app/admin/user.rb

```diff
@@ -2,6 +2,10 @@
   menu :priority => 6
   actions :all
 
+  before_create do |user|
+    user.send_request_to_patron_creator_service if user.school.present?
```

Comment:
I'd like to see a code comment, explaining what this line is for, and what the correct path is, if school isn't present.